### DISCUSSION TITLE
Command Starting with a Number

### DISCUSSION THREAD

**Discussion by bnerickson** [amoffat/sh#721] [Q&A] [answered]
I'm trying to execute some 7z commands using sh, but python doesn't accept variables starting with a number:

```
>>> import sh
>>> from sh import 7z
  File "<stdin>", line 1
    from sh import 7z
                   ^
SyntaxError: invalid syntax
```

Is there a workaround for this?

**Comment by amoffat** [comment, marked as answer]
Try instantiating the Command object manually:

```python
import sh
seven_z = sh.Command("7z")
seven_z(...)
```

https://sh.readthedocs.io/en/latest/sections/command_class.html#Command

**Reply by bnerickson** [reply]
That did it.  Thanks!
